src/main/java/com/sarahhand/fractals/juliaconstants/ConstantsPanel.java

```diff
@@ -0,0 +1,51 @@
+package com.sarahhand.fractals.juliaconstants;
+
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JTextField;
+
+/** Extension from JPanel that adds a JLabel and a JTextField to itself.
```

Comment:
Can you add additional comments to the Javadoc for what the class does - what's the purpose of the class?